packages/curve-ui-kit/src/features/connect-wallet/ui/ConnectedWalletLabel.tsx

```diff
@@ -16,7 +16,7 @@ export const ConnectedWalletLabel: FunctionComponent<ConnectedWalletLabelProps>
   onDisconnectWallet,
   ...props
 }) => (
-  <Button size="small" color="ghost" onClick={onDisconnectWallet} {...props}>
-    <AddressLabel value={walletAddress} />
+  <Button size="small" color="ghost" onClick={onDisconnectWallet} title={walletAddress} {...props}>
```

Comment:
Title is nice, but a [tooltip](https://mui.com/material-ui/react-tooltip/) is probably nicer

Reply:
I'm not even sure we need a tooltip or title at all. This is the wallet connect button that shows an abbreviated address once connected; I've never seen it before where a tooltip shows the entire address on hover. Imo it should be removed and if the user wants to copy their address they should be able to in the upcoming user profile menu where it's displayed on top as a big header.

Reply:
The tooltip can be useful (imo), what's the drawback?

Reply:
It's multifaceted, the information displayed wouldn't lead to any meaningful action to the user or give information they already don't know. On top of that the address is relatively large, and so would the tooltip be; a bit distracting.